packages/appstore/src/components/wallet-cards-carousel/wallet-cards-carousel.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useRef } from 'react';
+import { ProgressBarOnboarding, Icon, Text, WalletCard } from '@deriv/components';
+import { getWalletHeaderButtons } from 'Constants/utils';
+import { TWalletAccount } from 'Types';
+import { formatMoney } from '@deriv/shared';
+import Slider from 'react-slick';
+import './wallet-cards-carousel.scss';
+
+interface WalletCardsCarouselProps {
+    readonly items: TWalletAccount[];
+}
+
+export const WalletCardsCarousel = ({ items }: WalletCardsCarouselProps) => {
+    const [activePage, setActivePage] = React.useState(0);
+    const ref = useRef<Slider | null>(null);
```

Comment:
```suggestion
    const slider_ref = React.useRef<Slider | null>(null);
```
naming it this way would make it easier to know this is a ref for an external library